app/controllers/videos_controller.rb

```diff
@@ -34,25 +34,19 @@ def new
   end
 
   def create
-    if owner_or_admin

```

Comment:
Where did this authorization check move to?

Reply:
Good catch, I though tt was already there in `before_filter :authenticate_user!, :only=>[:new,:create]` but that doesn't do the same thing.  I removed this to get more test coverage.  I'll add it back.

Reply:
Sorry, I was right - I picked the wrong filter when I looked after your comment.

Currently all routes are filtered through:

`before_filter :authenticate_owner!`  (line 7)

which does:

```
  def authenticate_owner!
    unless owner_or_admin
      redirect_to(root_path, :notice => "You aren't authorized for this action.")
    end
  end
```

I removed the `if` in the method because the `else` clause was unreachable for code coverage.

Reply:
This change is covered by this unit test:

https://github.com/concord-consortium/video-paper-builder/blob/8811440a37acfbc457d1fe2246c9a0fd536fd9b3/spec/controllers/videos_controller_spec.rb#L69-L75